src/Umbraco.Web.UI.Client/src/views/components/forms/umb-checkbox.html

```diff
@@ -16,5 +16,12 @@
         </span>
     </span>
 
+    <i class="{{vm.iconClass}}" style="margin-left: 26px;" aria-hidden="true"></i>
+
     <span class="umb-form-check__text">{{vm.text}}</span>
 </label>
+
+
+<!-- TODO
```

Comment:
Is this note needed here as you seem to have handled this in a further commit?

Reply:
Oops! Removed now :-)